e2e/README.md

```diff
@@ -91,4 +91,8 @@ The e2e robot test framework includes 4 layers:
  ---------------------------------------------------------------------
 ```
 
- __* Each layer can only call functions from the next layer or the same layer. Skip-layer is strictly forbidden. For example, Keyword Definition layer can only call functions in Keyword Implementation layer or Keyword Definition layer, directly call functions in Basic operations layer is strictly forbidden.__
\ No newline at end of file
+ __* Each layer can only call functions from the next layer or the same layer. Skip-layer is strictly forbidden. For example, Keyword Definition layer can only call functions in Keyword Implementation layer or Keyword Definition layer, directly call functions in Basic operations layer is strictly forbidden.__
+
+### Convention
+
+Use the `ResouceCache` to storing shared data, allowing other keywords to reuse this information. This approach simplifies keyword complexity and minimizes redundant API calls needed to fetch resources.
```

Comment:
Thanks @c3y1huang for adding it, I was hoping we can add all the conventions we are using for writing test cases with robot framework in general.

1. Use 'Gherkin` format for writing test cases.
2. Use the `ResouceCache` to storing shared data ... 
3. etc

Reply:
I'd suggest having another issue/PR for the convention enhancements to avoid overcomplicating the purpose of this one. WDYT?

Reply:
Sounds good.

Reply:
https://github.com/longhorn/longhorn/issues/7272